saleor/graphql/webhook/utils.py

```diff
@@ -0,0 +1,21 @@
+from graphene.utils.str_converters import to_snake_case
+from graphql import parse, print_ast
+
+
+def get_event_type_from_subscription(query: str):
```

Comment:
Do you think, the function is robust enough?

Reply:
We need to check if it works well with fragments (I think it does, but worth checking) e.g:

```
fragment OrderFragment on Order {
  id
	number
  lines {
    id
  }
}

subscription {
  event {
    ...on OrderCreated {
      order {
        ...OrderFragment
      }
    }
  }
}
```

Reply:
We could also cover the case like:
```graphql
fragment NotUsedEvents on Event {
  ... on OrderCreated {
    order {
      id
    }
  }
}

subscription {
  event {
    ... on OrderUpdated {
      order {
        id
      }
    }
  }
}
```
 or the case when `event` is a fragment
 
 ```graphql
 fragment EventFragment on Event{
  ...
}
subscription {
  event{
   ...EventFragment    
  }
}

```